frontend/lib/norent/homepage.tsx

```diff
@@ -135,6 +134,16 @@ export const NorentHomePage: React.FC<{}> = () => (
           <br />
           <BuildMyLetterButton isHiddenMobile />
           <br />
+          <p className="is-size-6">
+            Made by non-profit{" "}
+            <OutboundLink
+              href="https://www.justfix.nyc/"
```

Comment:
Since it's being repeated a number of times and we might want to eventually use it elsewhere on the site, it might be useful to factor this out into a `JUSTFIX_WEBSITE_URL` constant or something.  Up to you tho, it's also easy enough to just do a find/replace later.

Reply:
Do you think we should do this globally (throughout the Tenant Platform) for all urls that link to the Org Site?

Reply:
Logging this as issue #1245 to continue the conversation.